consent.yaml

```diff
@@ -185,6 +185,260 @@ paths:
+                        dataHolderName: ANZ
+                        dataHolderBrandId: f632041c-d7c8-4679-a165-aa406cd62b13
+                        dataHolderLogoUri: 'https://www.abank.com.au/logo'
+                        useCase:
+                          id: HOME_LOAN
+                          name: Home Loan Assessment
+                          active: true
+                          softwareProductId: software product id
+                          description: Data will be used to assess your eligibility for a home loan.
+                          priority: 1
+                          historicalCollectionPeriodInDays: 90
+                          notificationType: EMAIL
+                          scopes:
+                            - name: 'Bank account name, type and balance'
+                              id: 'bank:accounts.basic:read'
+                              purpose: This will allow us to provide best of kind service
+                              claims:
+                                - Name of account
+                                - Type of account
+                                - Account balance
+                              priority: 1
+                          accessFrequency: ONCE_OFF
+                          sharingDurations:
+                            - ONE_YEAR
+                          dataHolders:
+                            - dataHolderBrandId: 7ff47a14-28bd-4b04-b216-67b367713a38
+                              brandName: Westpac
+                              logoUri: 'https://www.westpac.com.au/logo'
+                              favourite: false
+                          features:
+                            - string
+                          industries:
+                            - BANKING
+                          osps:
+                            - providerName: Adatree
+                              serviceDescription: Adatree's Data Recipient Platform is a turnkey SaaS solution providing CDR as a Service.
+                              accreditationId: ADRBNK000071
+                              cdrPolicyUri: 'https://adatree.com.au/cdr-policy'
+                          consumerType: INDIVIDUAL
+                          customDataSchema:
+                            $schema: 'https://json-schema.org/draft/2019-09/schema'
+                            type: object
+                            title: custom data Schema
+                            description: JSON schema of customer data
+                            purpose: explain why custom data is needed
+                            properties:
+                              employerName:
+                                type: string
+                                displayText: Employer name
+                                uiComponent:
+                                  inputType: text
+                                rules:
+                                  required: Employer name is required
```

Comment:
this example is showing up as an error for me in stoplight:
![image](https://github.com/user-attachments/assets/365ebdcf-38df-40ff-b6cd-e282767c7fcf)

our model says it should be a `string` type but the example shows it's an object:
![image](https://github.com/user-attachments/assets/4e5f66da-f1bd-442e-b0f8-09a5c3999744)

Reply:
Having a look now, it is so hard to please everyone 🤣  
https://validator.swagger.io was happy with https://raw.githubusercontent.com/Adatree/api-specs/97788fc6ebe47b6baa598453a672e7345461cc9d/consent.yaml

Reply:
i'm guessing `validator.swagger.io` didn't validate the example against the model 🤣

Reply:
is it fixed now?